Formula/mopidy-local-sqlite.rb

```diff
@@ -0,0 +1,27 @@
+class MopidyLocalSqlite < Formula
+  desc "Mopidy extension for playing music from local sqlite database"
+  homepage "https://github.com/mopidy/mopidy-local-sqlite"
+  url "https://files.pythonhosted.org/packages/0f/4d/f156196381ae8cea81bd3b75a493a180a137f01a568aacb77f0f5e223c08/Mopidy-Local-SQLite-1.0.0.tar.gz"
+  sha256 "5207b92e049b8ec60825f18065382d46eacb9dd328cf1b227f67e6771efefc13"
+  head "https://github.com/mopidy/mopidy-local-sqlite.git"
+  revision 1
```

Comment:
Remove. See explanation in the other formula.

```suggestion
```

Reply:
Done !